lib/operators/input.coffee

```diff
@@ -24,6 +24,8 @@ class Insert extends Operator
           @typedText,
           normalizeLineEndings: true
         )
+        cursor = @editor.getLastCursor()
+        cursor.moveLeft() unless cursor.isAtBeginningOfLine()
```

Comment:
This was because line 23 uses getCursorBufferPosition which only deals with the last cursor. If that gets changed, my two lines would want to be updated too. 

(I'm still new here so I go by existing code – I haven't even used multiple cursor much yet, and never in vim-mode.)